### DISCUSSION TITLE
Can I use "react-query" as part of my library name?

### DISCUSSION THREAD

**Discussion by iamchanii** [TanStack/query#3005] [Q&A] [answered]
Hi there! at the first, thanks to all contributors of react-query. I can save lots of my days from your works.

I've been using react-query for 2-3 years and recently, I built a library to help use react-query more efficiently and used it for my job project. it was helpful for me and my team, so I decided to make it open-source!

Now I have a question from the moment I decided to name it "react-query-helper". Can I use "react-query" as part of my library name? because someone can find my library from a search in npm by searching "react-query" and they should get some curious...

So I want to check two points: 1. Can I use it as part of the library name? 2. If not so, can you have any suggestions for me? 😉 

Thank you. have a nice day!

**Comment by tannerlinsley** [comment, marked as answer]
Sure, I don't see why not.

**Reply by iamchanii** [reply]
So lovely! thank you so much!
